CONTRIBUTING.md

```diff
@@ -152,6 +152,9 @@ This is a small overview of the contents of this repository:
 
 > [!NOTE]
 > All Camunda core modules are built and tested with JDK 21. Most modules use language level 21, exceptions are: camunda-client-java, camunda-process-test-java, zeebe-bpmn-model, zeebe-build-tools, camunda-client-java, zeebe-gateway-protocol zeebe-gateway-protocol-impl, zeebe-protocol, and zeebe-protocol-jackson which use language level 8.
+>
+> [!NOTE]
+> To avoid concurrent Maven execution issues, ensure the project is only opened in one IDE at a time while running Maven build commands (e.g., `./mvnw clean install`). Opening the same project in multiple IDEs simultaneously can cause build failures due to conflicting Maven tasks.
```

Comment:
🙈  Ups I somehow missed that you wanted to start a new note.

Looks like this is not working

<img width="842" height="394" alt="Image" src="https://github.com/user-attachments/assets/bb2a9db0-0747-4bd5-a986-f86ef52cfe8a" />



---



Feel free to remove the `>` again to have a separate note or remove the `![NOTE]`